eng/packaging.targets

```diff
@@ -312,4 +317,10 @@
     <Error Condition="'$(ServicingVersion)' == '0'" Text="ServicingVersion is set to 0 and it should be an increment of the patch version from the last released package." />
   </Target>
 
+  <Target Name="ValidatePackageReadmeExists"
```

Comment:
I suppose this is all relative to the project that is invoking the target, hence why README.md does not really need a prefix path?

Reply:
Yes. That path in MSBuild is relative to the project file.